packages/elements/src/collapse/index.ts

```diff
@@ -195,15 +202,17 @@ export class Collapse extends BasicElement {
    */
   protected render (): TemplateResult {
     return html`
-      <ef-header part="header" level="${this.level}" @tap="${this.handleTap}">
-        <ef-icon icon="right" slot="left" part="toggle"></ef-icon>
+      <ef-header part="header" level="${this.level}" @tap=${this.handleTap} role="heading" aria-level=${this.ariaLevel}>
+        <div part="header-toggle" id="header-toggle" role="button" tabindex="0" aria-expanded="${this.expanded}" aria-controls="content">
+          ${this.header}
+        </div>
+        <ef-icon icon="right" slot="left" part="toggle" aria-hidden="true"></ef-icon>
```

Comment:
Why `aria-hidden`? Icon is not focusable by default so should not be visible by the screen reader.

Reply:
For disabled people, it is not necessary to focus and navigate to the icon because you can use the entire header as a button to toggle, and the screen reader announces `expanded/collapsed` every time when focusing on the header that is the same meaning as the icons.

Reply:
How can you focus on `ef-icon`?

Reply:
Actually, it is not a focus, but it is something like an `current element` for a screen reader (you can see a border around of active element).  if you use the `VoiceOver` on macOS you can use the keyboard command `Capslock + ShiftUp + Down Arrow` to navigate to inside the header content.

Reply:
Yes, the user can, but why is it a problem? If you can see the icon, the impaired user might want to see it as well via keyboard commands. You can set `aria-label` for the icon in that case

Reply:
The problem is if no `aria-label` for the icon the screen reader will announce the heading with an icon that is not good because it is very long and make a user confused.
For example, it announces `"Heading level 2, 2 items, icon right image, Title xxx, expanded button"` instead of `"Heading level 2, Title xxx, expanded button"`